modules/recruiting/ghost/variables.tf

```diff
@@ -0,0 +1,62 @@
+variable "env_name" {
+  type = string
+}
+
+variable "resource_group_name" {
+  type = string
+}
+
+variable "resource_group_location" {
+  type = string
+}
+
+/* MySQL */
+variable "mysql_fqdn" {
+  type = string
+}
+
+variable "mysql_connection_user" {
```

Comment:
```suggestion
variable "mysql_username" {
```

Reply:
Done in https://github.com/Tietokilta/tietokilta.fi-infra/pull/12/commits/51fe870d36f230e6b80a6c19f46f3539ee195c39